Makefile

```
@@ -50,6 +50,14 @@ lint-html:  ## Run HTML Linters
 lint-frontend:  ## Run front-end linters
 	npm run lint
 
+.PHONY: lint-migrations
+lint-migrations: ## Run django-migration-linter
+	poetry run python manage.py lintmigrations
+
+.PHONY: lint-migrations-custom
+lint-migrations-custom: ## Run django-migration-linter
+	cd scripts && poetry run python lintmigrations_custom.py
```

```suggestion
	poetry run python scripts/lintmigrations_custom.py
```

with the change suggestions in `scripts/lintmigrations_custom.py` below

---

We tried this before but we didn't have `BASE_DIR = Path(__file__).parent.parent` and `BASE_DIR / "manage.py"`😅- all working now! Thank you 🎉